<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgMainWindow</class>
 <widget class="QDialog" name="dlgMainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>560</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>488</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>488</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Belenus Admin</string>
  </property>
  <widget class="QLabel" name="lblAdminPassword">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Password :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledAdminPassword">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>230</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText</set>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pbExitApplication">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>530</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit application</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSerialList">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Registered Serial numbers :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSerialNumber">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Serial number :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledSerialNumber">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>210</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="pbRegisterSerial">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listSerialNumbers">
   <property name="geometry">
    <rect>
     <x>259</x>
     <y>30</y>
     <width>211</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbEditSerial">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDeleteSerial">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSerialInfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>230</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Info:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbEmptySerial">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>20</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>470</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>470</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>3</width>
     <height>250</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>3</width>
     <height>250</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblBelenusServer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Belenus Server</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledAddress">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>000.000.000.000; </string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPort">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledPort">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>00000; </string>
   </property>
   <property name="maxLength">
    <number>5</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pbConnect">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblConnectionStatus">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Not connected</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>470</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>3</width>
     <height>40</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>260</y>
     <width>3</width>
     <height>40</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listStatus">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>410</y>
     <width>471</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblCustomMessage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Custom message body :</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbCON">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>CON</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbREQ">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>REQ</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbPSW">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PSW</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbARC">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ARC</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbSQL">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SQL</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbPRC">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>310</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PRC</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbBYE">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>310</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>BYE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbRegisterSerialOnServer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Register on server</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDeleteSerialFromServer">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete from server</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSendCustom">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>120</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send custom message</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbCustomMsgBody">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>470</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbINF">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>INF</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbERR">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>310</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ERR</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>ledAdminPassword</tabstop>
  <tabstop>ledSerialNumber</tabstop>
  <tabstop>pbRegisterSerial</tabstop>
  <tabstop>pbExitApplication</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
